httpbp/errors.go

```diff
@@ -671,6 +671,38 @@ func (ce ClientError) RetryAfterDuration() time.Duration {
 	return ce.RetryAfter
 }
 
+// Retryable implements retrybp.RetryableError.
+//
+// It returns no decision if the status code is unknown (0),
+// true on any of the following conditions and false otherwise:
+//
+// - There was a valid Retry-After header in the response
+//
+// - The status code was 425 (too early) or 429 (too many requests)
+//
+// - The status code was 5xx
+func (ce ClientError) Retryable() int {
```

Comment:
note: this implementation is slightly different from the default retryable codes used by `thriftbp.BaseplateErrorFilter` (https://github.com/reddit/baseplate.go/blob/c737670824b8c308b2ccc49847810bcbf2dd098f/thriftbp/errors.go#L41-L47). Not sure if we want to make them consistent.

Reply:
Hmm, I think we want to at least add TOO_MANY_REQUESTS to this https://github.com/reddit/baseplate.py/blob/7460e89d70497c42ee4ea95a243f01c421b62125/baseplate/thrift/baseplate.thrift#L155

We also _might_ want to add INTERNAL_SERVER_ERROR, BAD_GATEWAY, and maybe INSUFFICIENT_STORAGE?

Reply:
:+1:

Made both to include the same 7 codes.